provider_packages/DEV-README.md

```diff
@@ -171,3 +171,124 @@ the version in PyPI does not contain the leading 0s in version name - therefore
 also do not container the leading 0s.
 
 * You can install the .whl packages with `pip install <PACKAGE_FILE>`
+
+
+# Testing provider package scripts
+
+The backport packages importing and tests execute within the "CI" environment of Airflow -the
+same image that is used by Breeze. They however require special mounts (no
+sources of Airflow mounted to it) and possibility to install all extras and packages in order to test
+importability of all the packages. It is rather simple but requires some semi-automated process:
+
+## Backport packages
+
+1. Prepare backport packages
+
+
+```shell script
+./breeze --backports prepare-provider-packages
+```
+
+This prepares all backport packages in the "dist" folder
+
+2. Enter the container:
+
+```shell script
+export INSTALL_AIRFLOW_VERSION=1.10.12
+export BACKPORT_PACKAGES="true"
+
+./scripts/ci/provider_packages/ci_enter_breeze_provider_package_tests.sh
```

Comment:
This script is wrongly named -- it isn't used by CI so shouldn't live under `scripts/ci`/

Reply:
Yep. It should be in dev/providers_packages.